core/test_network/src/lib.rs

```diff
@@ -221,6 +209,7 @@ where
     ) -> (Self, Client) {
         let mut configuration = Configuration::test();
         configuration.queue.maximum_transactions_in_block = max_txs_in_block;
+        configuration.logger.max_log_level = iroha_logger::Level::INFO.into();
```

Comment:
Really interested to see if we can set this somewhere else, so it's easier to refactor.

Reply:
Maybe `impl Default`